### PR TITLE
Added state wise deltas

### PR DESCRIPTION
**Pull request by sumitmckv**
Required for https://github.com/covid19india/covid19india-react/issues/12
state_wise_deltas

### CODE REVIEW

**Comment by rito-rapido** [comment]
Can we get this out there ASAP ? Could really help in building the front end. Working on it! :)

File: sheet-to-json.js [hunk 1]
```diff
@@ -50,12 +53,37 @@ function sortObjByKey(value) {
     value;
 }
 
+function getDelta(state) {
+  return  rawData.raw_data.reduce((stat, row) => {
+    let stateName = row.detectedstate;
+    let isToday = moment().isSame(moment(row.dateannounced, "DD-MM-YYYY"), "day");
+    if (stateName && (stateName === state || state === "Total") && isToday) {
+      let currentStatus = row.currentstatus;
+      if (currentStatus) {
```
Comment (anuragjain) [inline sheet-to-json.js]: put a console.error for if currentStatus is empty or not one of the three in the sheet, 
We will know that sheet needs to be corrected basis this log.

File: sheet-to-json.js [hunk 2]
```diff
@@ -50,12 +53,37 @@ function sortObjByKey(value) {
...
+        stat.confirmed += 1;
+        switch (currentStatus) {
```
Comment (jkandasa) [inline sheet-to-json.js]: @sumitmckv I see the following status:
```
Recovered
Hospitalized
Deceased
Migrated
```
Include `Migrated` as well. This field should be included on https://api.covid19india.org/data.json `statewise` too

Comment (anuragjain) [reply sheet-to-json.js]: what does Migrated signify? 
The sum of recovered, hospitalized and deceased should be total confirmed cases. Introduction of Migrated will have that maths messed up.
Also the main table on website shoes confirmed, active, recovered and deceased. Not sure which bucket the migrated data will fall into.

Comment (jkandasa) [reply sheet-to-json.js]: I do not know about `Migrated` status. I see this status on the `raw_data.json`.
So thought of including it.

```json
{
  "agebracket": "",
  "backupnotes": "Travelled to Singapore (Feb) and Canada (Mar)",
  "contractedfromwhichpatientsuspected": "",
  "currentstatus": "Migrated",
  "dateannounced": "17/03/2020",
  "detectedcity": "Saket",
  "detecteddistrict": "",
  "detectedstate": "Delhi",
  "estimatedonsetdate": "",
  "gender": "",
  "nationality": "India",
  "notes": "Travelled from Singapore (Feb) and Canada (Mar)",
  "patientnumber": "130",
  "source1": "http://health.delhigovt.nic.in/wps/wcm/connect/a50f1c804d973055ae77ff982ee7a5c7/HB17.pdf?MOD=AJPERES&lmod=-333510648&CACHEID=a50f1c804d973055ae77ff982ee7a5c7",
  "source2": "https://www.hindustantimes.com/delhi-news/delhi-s-8th-covid-19-case-flew-to-singapore-without-waiting-for-test-result/story-c4YFWwviCvti3TYnA124SM.html",
  "source3": "",
  "statepatientnumber": "DL-P8",
  "statuschangedate": "20/03/2020"
}
```

File: sheet-to-json.js (hunk 1 above)
Comment (sumitmckv) [reply sheet-to-json.js]: @sumitmckv added empty check for currentStatus

**Comment by JunaidBabu** [comment]
@sumitmckv this is updating based on UTC I think. Can you change it to IST?

**Comment by sumitmckv** [comment]
@JunaidBabu will create PR for this today.

**Pull request merged**